### PR TITLE
Update findings page documentation

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-app/findings.md [hunk 1]
```diff
@@ -130,36 +131,67 @@ To manage, change, open or ignore findings, follow the the triage processes desc
...
+
+##### Ignoring individual findings
+
+To **ignore individual findings**, follow these steps:
+
+1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, click the **Status** filter, and then select **Open** toggle to see all open findings.
+1. Next to a finding you want to ignore, click the **Ignore** <i class="fa-regular fa-chevron-down"></i>.
+1. Optional: Select a reason of why you are ignoring a finding. Choose either: **False positive**, **Acceptable risk**, **No time to fix**
+1. Click **Save**.
+1. Optional: Select whether you want to ignore all findings in **Just this file**, **This directory**, or **Parent directory**.
```
Comment (sabrinabrogren) [inline docs/semgrep-app/findings.md:157]: Should we explain what this does (which is add the selected path to the .semgrepignore)?

**Comment by sabrinabrogren** [comment]
looks good to me, just left one comment! Also I can follow up in the PR when this feature is merged?

**Review by pabloest — APPROVED** [review]

**Comment by sabrinabrogren** [comment]
Feature has merged! Should I merge this PR?

**Comment by pabloest** [comment]
@sabrinabrogren Yeah. Adam said he can implement suggestions when he is back.

**Pull request merged**